gnmvidispine/vs_notifications.py

```diff
@@ -42,7 +42,7 @@ def _safe_set_nodecontent(self, xpath, newval, parent=None):
         from xml.etree.cElementTree import SubElement
         if parent is None:
             parent = self.dataContent
-        if not isinstance(newval,basestring):
+        if not isinstance(newval,str):
             newval = str(newval)
```

Comment:
again, some kind of safe_string function might be better here, although this looks like it's a catch-all to convert e.g. numbers into strings pre-serialization too